lms/djangoapps/course_api/tests/test_forms.py

```diff
@@ -0,0 +1,144 @@
+"""
+Tests for Course API forms.
+"""
+
+import ddt
+from django.contrib.auth.models import AnonymousUser
+from django.http import QueryDict
+from itertools import product
+from urllib import urlencode
+
+from openedx.core.djangoapps.util.test_forms import FormTestMixin
+from student.tests.factories import UserFactory
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+from ..forms import CourseDetailGetForm, CourseListGetForm
+
```

Comment:
Relative imports for within a django app should be perfectly fine.  Have I missed something?